app/CocoaPods/CPSplashRecentDocumentsController.h

```diff
@@ -0,0 +1,24 @@
+//
+//  CPSplashRecentDocumentsController.h
+//  CocoaPods
+//
+//  Created by Orta Therox on 05/09/2015.
+//  Copyright (c) 2015 CocoaPods. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <Cocoa/Cocoa.h>
+
+/// Handles the getting the recent documents from
+/// the shared NSDocumentController
+
+@interface CPSplashRecentDocumentsController : NSObject
+
+/// Used in NSTableView via IB bindings
+@property (readwrite, nonatomic, copy) NSArray *recentDocuments;
```

Comment:
**NIT:** *Inconsistency in order of property parameters. Let's put `nonatomic` in a consistent position.*

Above we have `(nonatomic, strong, readonly)`:

```objc
@property (nonatomic, strong, readonly) CPSplashWindowController *splashWindowController;
```

Here we have `(readwrite, nonatomic, copy)`, please change to `(nonatomic ,readwrite, copy)`